src/main/java/org/helpberkeley/memberdata/RestaurantTemplateParser.java

```diff
@@ -224,6 +228,10 @@ private void auditControlBlockRow(RestaurantBean bean) {
         String directive = bean.getControlBlockDirective();
 
         switch (directive) {
+            case Constants.CONTROL_BLOCK_FORMULA:
+                hasFormulaDirective = true;
+                auditControlBlockFormula(bean);
```

Comment:
Update this to append to errors.  Update auditControlBlockFormula() to return a String of errors ("" for none).